app/main.dev.js

```diff
@@ -54,45 +54,80 @@ app.on('window-all-closed', () => {
   // Respect the OSX convention of having the application in memory even
   // after all windows have been closed
   if (process.platform !== 'darwin') {
-    app.quit();
+    app.quit()
   }
-});
+})
 
 
 app.on('ready', async () => {
   if (process.env.NODE_ENV === 'development' || process.env.DEBUG_PROD === 'true') {
-    await installExtensions();
+    await installExtensions()
   }
 
   mainWindow = new BrowserWindow({
     show: false,
     frame: false
-  });
+  })
 
-  mainWindow.maximize();
+  mainWindow.maximize()
 
-  mainWindow.loadURL(`file://${__dirname}/app.html`);
+  mainWindow.loadURL(`file://${__dirname}/app.html`)
 
   // @TODO: Use 'ready-to-show' event
   //        https://github.com/electron/electron/blob/master/docs/api/browser-window.md#using-ready-to-show-event
   mainWindow.webContents.on('did-finish-load', () => {
     if (!mainWindow) {
-      throw new Error('"mainWindow" is not defined');
+      throw new Error('"mainWindow" is not defined')
     }
-    
-    mainWindow.show();
-    mainWindow.focus();
-  });
 
-  mainWindow.on('closed', () => {
-    mainWindow = null;
-  });
+    mainWindow.show()
+    mainWindow.focus()
+  })
 
-  const menuBuilder = new MenuBuilder(mainWindow);
-  menuBuilder.buildMenu();
-});
+  mainWindow.on('closed', () => {
+    mainWindow = null
+  })
+
+  const menuBuilder = new MenuBuilder(mainWindow)
+  menuBuilder.buildMenu()
+
+
+  //Set ENV variables for go binaries
+  process.env.GOPATH = "~/gocode"
+  process.env.PATH = process.env.PATH + ":" + process.env.HOME + "/gocode/bin"
+  exec("chmod +x pre_run.sh & ./pre_run.sh")
+
+  // Check to see if LND is running. If not, start it.
+  lookup({ command: 'lnd' }, (err, results) => {
+    if (err) { throw new Error( err ) }
+
+    if (!results.length) {
+      // Alert user that LND is starting
+      console.log('STARTING LND')
+
+      // Start LND
+      neutrino = spawn(
+        'lnd',
+        [
+          '--bitcoin.active',
```

Comment:
I've been using lnd without the `--bitcoin.active` flag, just `--bitcoin.testnet`. After perusing this PR, I decided to try adding the bitcoin.active flag. It makes my previous onchain (testnet) tx-history disappear because I think it uses a different folder for the wallet.dat.

Reply:
Do you have your lnd updated ?